README.md

```diff
@@ -59,6 +59,28 @@ from other services too: you'll just need to provide your file URLs for import.
                                   check requests.
                                   
   ```              
+ 
+## Output file format
+The output file format is quite simple.
+For each input file URL of Filestack file handle, there are 3 columns divided by the \t symbol (tab).
+
+The first column holds input file URL or Filestack handle itself, 
+second column — upload operation status: success or failed, 
```

Comment:
Nitpick: that's not quite right. Something could have either succeeded or failed. Something can be either a success or a failure/fail. The mix of those two is somewhat weird. However, this could work fine.